frontend/src/components/site/DosaEditorial.module.css

```
@@ -0,0 +1,514 @@
+.home,
+.services {
+  --editorial-max: 1440px;
+}
+
+@media (min-width: 901px) {
+  .home :global(.hero-orientation) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)